plugin/executor/containerd/containerd.go

```diff
@@ -98,23 +98,23 @@ func (e *Executor) Create(id string, spec specs.Spec, stdout, stderr io.WriteClo
 }
 
 // Restore restores a container
-func (e *Executor) Restore(id string, stdout, stderr io.WriteCloser) error {
+func (e *Executor) Restore(id string, stdout, stderr io.WriteCloser) (bool, error) {
 	alive, _, err := e.client.Restore(context.Background(), id, attachStreamsFunc(stdout, stderr))
 	if err != nil && !errdefs.IsNotFound(err) {
-		return err
+		return false, err
 	}
 	if !alive {
 		_, _, err = e.client.DeleteTask(context.Background(), id)
 		if err != nil && !errdefs.IsNotFound(err) {
-			logrus.WithError(err).Errorf("failed to delete container plugin %s task from containerd", id)
+			logrus.WithError(err).WithField("id", id).Errorf("failed to delete plugin container task from containerd")
```

Comment:
`plugin container task` is a bit much :) Can we stick to `plugin task`?

Reply:
Well, there's the task and the actual container obkect. Which both have to be dealt with.

Reply:
It may be better to add "plugin" in Withfield and remove it from the actual error message then?

Reply:
I think I read this from my phone before... `plugin task` works.